### PR TITLE
Eng 8575 no java heap warning for most standard configurations - better wording in others

### PR DESCRIPTION
**Pull request by JadenW**

### CODE REVIEW

File: src/frontend/org/voltdb/RealVoltDB.java [hunk 1]
```diff
@@ -2914,9 +2917,9 @@ private void checkHeapSanity(boolean isPro, int tableCount, int sitesPerHost, in
     {
         long megabytes = 1024 * 1024;
         long maxMemory = Runtime.getRuntime().maxMemory() / megabytes;
-        long drRqt = isPro ? 128 * sitesPerHost : 0;
+        // DRv2 now is off heap
+        // long drRqt = isPro ? 128 * sitesPerHost : 0;
```
Comment (jhugg) [inline src/frontend/org/voltdb/RealVoltDB.java]: Don't comment out code. Just remove code that isn't used anymore. Add the comment in the commit message.

**Comment by manjujames** [comment]
This will show the warning message every time you update the catalog, regardless of whether new tables were added or not. Otherwise the changes look good.

File: src/frontend/org/voltdb/RealVoltDB.java (hunk 1 above)
Comment (JadenW) [inline src/frontend/org/voltdb/RealVoltDB.java]: Removed. thanks .

**Comment by JadenW** [comment]
Yes. Once the heap estimation is above the max heap avaible, this warning will continue to reappear to console log every time during UAC. It should disappear when the heap estimation drop below the threshold.

**Comment by JadenW** [comment]
Squashed commit

File: src/frontend/org/voltdb/RealVoltDB.java [hunk 2]
```diff
@@ -2914,9 +2917,8 @@ private void checkHeapSanity(boolean isPro, int tableCount, int sitesPerHost, in
     {
         long megabytes = 1024 * 1024;
         long maxMemory = Runtime.getRuntime().maxMemory() / megabytes;
-        long drRqt = isPro ? 128 * sitesPerHost : 0;
+        // DRv2 now is off heap
```
Comment (jhugg) [inline src/frontend/org/voltdb/RealVoltDB.java]: What does this comment add here? It only makes sense if you knew the old code.

**Pull request closed without merging**